### PR TITLE
Pinterest Ads source documentation

### PR DESCRIPTION
**Pull request by mohan-h**
Initial commit

### CODE REVIEW

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+>[!NOTE]
+>
+>The [!DNL Pinterest Ads] source is in beta. See the [Sources overview](../../home.md#terms-and-conditions) for more information on using beta-labelled connectors.
+
+Adobe Experience Platform allows data to be ingested from external sources while providing you with the ability to structure, label, and enhance incoming data using [!DNL Platform] services. You can ingest data from a variety of sources such as Adobe applications, cloud-based storage, databases, and many others.
+
+[!DNL Experience Platform] provides support for ingesting data from a third-party advertising system. Support for advertising providers include [!DNL Pinterest Ads].
+
+[[!DNL Pinterest Ads]](https://ads.pinterest.com/) allow marketers to reach 400 million people who use [[!DNL Pinterest]](https://www.pinterest.com), a visual discovery engine for finding ideas like recipes, home and style inspiration, and more, on the internet using images, and on a smaller scale, animated GIFs and videos, in the form of pin boards.
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: Revision here so we can break this up to a few sentences:

"[!DNL Pinterest] is a visual discovery engine for finding recipes, home décor, style inspiration, and other ideas across the web. These are presented on a small scale using images, animated GIFs, and videos in pinboard format. [!DNL Pinterest Ads] allows you to grow your business and reach 400 million people using [!DNL Pinterest]."

Also, can we check on branding for "Pinterest Ads"? I'm seeing both upper and lower case for "Ads/ads" on their interface. Just want to confirm if the proper branded term is Pinterest Ads or Pinterest ads

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+[!DNL Experience Platform] provides support for ingesting data from a third-party advertising system. Support for advertising providers include [!DNL Pinterest Ads].
+
+[[!DNL Pinterest Ads]](https://ads.pinterest.com/) allow marketers to reach 400 million people who use [[!DNL Pinterest]](https://www.pinterest.com), a visual discovery engine for finding ideas like recipes, home and style inspiration, and more, on the internet using images, and on a smaller scale, animated GIFs and videos, in the form of pin boards. 
+
+By using [!DNL Pinterest Ads], which are typically just like any other pins, you can reach out to these users through targeted advertisements to discover and buy your products. Pins from [!DNL Pinterest Ads] are sponsored to receive extra exposure in relevant search results. Pinterest Business account holders can choose to promote existing best-performing pins, create a new image or video, or even promote an image that’s been pinned from a website. Pinterest offers several ad formats to help you meet your specific campaign goals.
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: "Business" should be lower case here. Alternatively, we can rephrase it to "Users subscribed to [!DNL Pinterest Business] can choose to...

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+## [!DNL Pinterest] APIs {#pinterest-apis}
+
+This Adobe Experience Platform [source](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [!DNL Pinterest] APIs to retrieve your [!DNL Pinterest Ads] data, along with all performance and metrics. The supported API endpoints are :
+* [Campaigns](https://developers.pinterest.com/docs/api/v5/#operation/campaigns/analytics) Analytics
+* [Ad Groups](https://developers.pinterest.com/docs/api/v5/#operation/ad_groups/analytics) Analytics
+* [Ads](https://developers.pinterest.com/docs/api/v5/#operation/ads/analytics) Analytics
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: Revise to:
* Campaign analytics
* Ad group analytics
* Ad analytics

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+The [!DNL Pinterest Ads] source connection allows you to bring the data from [!DNL Pinterest] to Experience Platform, where you can execute analytics.
+
+The data is returned from today for a backdated range of 186 days. Refer to the [Guardrails](#guardrails) section for details.
+
+[!DNL Pinterest Ads] uses bearer tokens as an authentication mechanism to communicate with the Pinterest APIs.
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: * Considering the context, it might be better if we revise this to: "The [!DNL Pinterest Ads] source integration allows you to..."
* "Data is returned from the current date up to a backdated range of 186 days. 
Let's also combine these three lines into one paragraph.

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+## Prerequisites {#prerequisites}
+
+The first step in creating a [!DNL Pinterest Ads] source connection is to ensure that you have a Pinterest developer account. If you do not have one already, see the [Signup](https://www.pinterest.com/business/create/?next=https://developers.pinterest.com/account-setup/) page to register and create your account.
+
+### Setup [!DNL Pinterest] App and generate Access Token {#create-app=and-generate-token}
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: * App should be lower cased "app" and "Access Token" should also be lower cased "access token"
* Typo in the anchor, "=" should be a hyphen "-"

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+>[!IMPORTANT]
+>
+> Generating the Access token on the UI provides limited access only to the scopes - pins:read, boards:read and user_accounts:read which is not adequate for usage with the Analytics [!DNL Pinterest] API endpoints.
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: Let's revise this to:
>[!IMPORTANT]
>
>It is recommended to use the APIs to generate your access token because generating your access token in the UI provides a limited access. Through the UI, you will only be able to access the following scopes: pins:read, boards:read and user_accounts:read. This limitation is not adequate for usage with the analytics endpoints of the [!DNL Pinterest] API.

Let me know if this is okay or if it loses the meaning we want to convey.

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+To generate this access token you will need to
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: let's update "this" to "your"

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+1. [Set up your App](https://developers.pinterest.com/docs/getting-started/set-up-app/).
+1. Follow the tutorial section to [Build a working authentication flow based on OAuth 2](https://developers.pinterest.com/docs/getting-started/authentication/).
+1. Obtain the required `Access Token`.
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: It looks like step 3 is part of the guide linked in step 2. In this case, we can restructure this to:

"To generate your access token, you must first [set up your app](https://developers.pinterest.com/docs/getting-started/set-up-app/) and then [authenticate using OAuth 2.0](https://developers.pinterest.com/docs/getting-started/authentication/)."

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+
+### Gather required credentials {#gather-required-credentials}
+
+In order to connect [!DNL Pinterest Ads] to Platform, you must provide values for the following connection properties:
+
+| Credential | Description |
+| --- | --- |
+| Access Token | The [!DNL Pinterest Ads] access token for your user account.<br> The token's user account must either be the Owner of the specified [!DNL Pinterest Ad] account, or have one of the necessary roles granted to them via Business Access: Admin, Analyst, Campaign Manager. Refer to the [Setup [!DNL Pinterest] App and generate Access Token](#create-app=and-generate-token) if you haven't already created one. |
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: * No need for the line break
* Let's revise "Owner" to lower case "owner"
* "Business Access: Admin, Analyst, or Campaign Manager"
* "For more information on the access token, please refer to the [[!DNL Pinterest] guide on generating your access token](https://developers.pinterest.com/docs/getting-started/set-up-app/)."

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+Note down your [!DNL Pinterest] `Ad Account ID`. Check the [Find IDs in Ads Manager](https://help.pinterest.com/en/business/article/find-ids-in-ads-manager) page for guidance on how to get the value.
+
+### Note [!DNL Pinterest] Camapign or Ad Group or Ad IDs {#gather-pinterest-ids}
+
+You will also need one or more [!DNL Pinterest] `Campaign` or `Ad Group` or `Ad` IDs corresponding to `Campaign` or `Ad Group` or `Ad` names, within the fore mentioned `Ad Account ID`, from which you want to retrieve the information. 
+
+>[!NOTE]
+>
+>[!DNL Pinterest] provides individual APIs to retrieve data associated with each ID, Accordingly you need to pass only the corresponding IDs for the ID type you are interested in.
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: Let's add these two to the credentials table:
| Credential | Description |
| --- | --- |
| Ad account ID | [Short description]. For information on retrieving your Ad account ID. Visit the [[!DNL Pinterest] guide on finding IDs in Ads Manager](https://help.pinterest.com/en/business/article/find-ids-in-ads-manager).
| Campaign, Ad group, or Ad ID | "The Campaign, Ad group, or Ad IDs that correspond with your Ad account ID. To obtain the required IDs, navigate to the [!DNL Pinterest] page for Pinterest Business Hub > Ad Account Summary > Campaigns / Ad groups / Ads and copy the required ID's mentioned just below each of their names."

Then let's have the same note after the credentials table. However, the note will break because it starts with a DNL, so let's revise to:
>The [!DNL Pinterest] API provides individual APIs to retrieve data associated with each ID. Therrefore, you need to pass only the corresponding IDs for the ID type you are interested in.

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+
+To obtain the required IDs, navigate to the [!DNL Pinterest] page for **Pinterest Business Hub** > **Ad Account Summary** > **Campaigns** / **Ad Groups** / **Ads** and copy the required ID's mentioned just below each of their names.
+
+# Guardrails {#guardrails}
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: Should be H2

## Guardrails

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+
+The time granularity supported by the [!DNL Pinterest] API are as below :
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: "The [!DNL Pinterest] API supports different time granularities for how data can be retrieved:"

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+* **TOTAL** - metrics are aggregated over the specified date range.
+* **DAY** - metrics are broken down daily.
+* **HOUR** - metrics are broken down hourly.
+* **WEEKLY** - metrics are broken down weekly.
+* **MONTHLY** - metrics are broken down monthly.
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: Let's put these in a table:

| Time granularity | Description |
| --- | --- |
|  **TOTAL** | The data metrics are aggregated over a specified date range. |
| **DAY** | The data metrics are broken down on a daily basis. |
| **HOUR** | The data metrics are broken down on an hourly basis.  |
| **WEEKLY** | The data metrics are broken down on a weekly basis. |
| **MONTHLY** | The data metrics are broken down on a monthly basis. |

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+
+The [API's](#pinterest-apis) leveraged for the [!DNL Pinterest Ads] source have time range restrictions as mentioned below :
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: "The time range restrictions for [!DNL Pinterest Ads] APIs are as follows:
* "If granularity is set to `HOUR`, then you can pull data from as far back as eight days before the current date in UTC time and the maximum time range support is three days."
* "If granularity is **not** set to `HOUR`, then you can pull data from as far bas 914 days before the current date in UTC time and the maximum time range supported is 186 days."

**Note**: I am a little confused by the second part of these two restrictions, what is the maximum time range for in this case?

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 15]
```diff
@@ -0,0 +1,100 @@
...
+* *If granularity is not HOUR, the furthest back you can are allowed to pull data is 914 days before the current date in UTC time and the max time range supported is 186 days.*
+* *If granularity is HOUR, the furthest back you can are allowed to pull data is 8 days before the current date in UTC time and the max time range supported is 3 days.*
+ 
+Based on the above [!DNL Pinterest] limitations the granularity for [!DNL Pinterest Ads] source is internally defined as **DAY**, this means the data retrieved will be limited to 186 days backdated from the current date. Accordingly you should
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: * Need a comma after "limitations"
* Second sentence is cut off

File: help/sources/connectors/advertising/pinterest-ads.md [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+* **WEEKLY** - metrics are broken down weekly.
+* **MONTHLY** - metrics are broken down monthly.
+
+The [API's](#pinterest-apis) leveraged for the [!DNL Pinterest Ads] source have time range restrictions as mentioned below :
+* *If granularity is not HOUR, the furthest back you can are allowed to pull data is 914 days before the current date in UTC time and the max time range supported is 186 days.*
+* *If granularity is HOUR, the furthest back you can are allowed to pull data is 8 days before the current date in UTC time and the max time range supported is 3 days.*
+ 
+Based on the above [!DNL Pinterest] limitations the granularity for [!DNL Pinterest Ads] source is internally defined as **DAY**, this means the data retrieved will be limited to 186 days backdated from the current date. Accordingly you should 
+
+Additionally refer to the [[!DNL Pinterest Rate Limits]](https://developers.pinterest.com/docs/reference/ratelimits/) documentation for the rate limits applicable to the underlying API calls. For example [!DNL Pinterest] imposes a rate limit of 1000 API calls daily on its API to read information from ads, ad groups or ad campaigns.
```
Comment (carlojavier) [inline help/sources/connectors/advertising/pinterest-ads.md]: Need a comma after additionally

File: help/sources/tutorials/api/create/advertising/pinterest-ads.md [hunk 17]
```diff
@@ -0,0 +1,879 @@
...
+* [Sources](../../../../home.md): Experience Platform allows data to be ingested from various sources while providing you with the ability to structure, label, and enhance incoming data using Platform services.
+* [Sandboxes](../../../../../sandboxes/home.md): Experience Platform provides virtual sandboxes which partition a single Platform instance into separate virtual environments to help develop and evolve digital experience applications.
+
+The following sections provide additional information that you will need to know in order to successfully connect to [!DNL Pinterest Ads] using the [!DNL Flow Service] API.
+
+## Prerequisites {#prerequisites}
+
+In order to connect [!DNL Pinterest Ads] to Experience Platform, you must provide values for the following connection properties:
+
+* The [!DNL Pinterest] `Access Token`.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/pinterest-ads.md]: lower and camel case: `accessToken`

File: help/sources/tutorials/api/create/advertising/pinterest-ads.md [hunk 18]
```diff
@@ -0,0 +1,879 @@
...
+curl -X POST \
+    'https://platform.adobe.io/data/foundation/flowservice/sourceConnections' \
+    -H 'Authorization: Bearer {ACCESS_TOKEN}' \
+    -H 'x-api-key: {API_KEY}' \
+    -H 'x-gw-ims-org-id: {ORG_ID}' \
+    -H 'x-sandbox-name: {SANDBOX_NAME}' \
+    -H 'Content-Type: application/json' \
+    -d '{
+        "name": "{YOURSOURCE} Source Connection",
+        "description": "{YOURSOURCE} Source Connection",
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/pinterest-ads.md]: Update placeholders with Pinterest Ads, same as the ones below

File: help/sources/tutorials/api/create/advertising/pinterest-ads.md [hunk 19]
```diff
@@ -0,0 +1,879 @@
...
+curl -X POST \
+    'https://platform.adobe.io/data/foundation/flowservice/targetConnections' \
+    -H 'Authorization: Bearer {ACCESS_TOKEN}' \
+    -H 'x-api-key: {API_KEY}' \
+    -H 'x-gw-ims-org-id: {ORG_ID}' \
+    -H 'x-sandbox-name: {SANDBOX_NAME}' \
+    -H 'Content-Type: application/json' \
+    -d '{
+        "name": "{YOURSOURCE} Target Connection",
+        "description": "{YOURSOURCE} Target Connection",
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/pinterest-ads.md:624]: Note the placeholders that need to be updated

File: help/sources/tutorials/api/create/advertising/pinterest-ads.md [hunk 20]
```diff
@@ -0,0 +1,879 @@
...
+```shell
+curl -X POST \
+    'https://platform.adobe.io/data/foundation/flowservice/flows' \
+    -H 'x-api-key: {API_KEY}' \
+    -H 'x-gw-ims-org-id: {ORG_ID}' \
+    -H 'x-sandbox-name: {SANDBOX_NAME}' \
+    -H 'Content-Type: application/json' \
+    -d '{
+        "name": "{YOURSOURCE} dataflow",
+        "description": "{YOURSOURCE} dataflow",
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/pinterest-ads.md:804]: two more placeholders here

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 21]
```diff
@@ -0,0 +1,264 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+title: Create a Pinterest Ads source connection in the UI
+description: Learn how to create a Pinterest Ads source connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+---
+# Beta Create a [!DNL Pinterest Ads] source connection in the UI
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md]: We can remove "Beta" from the heading and add badge: "Beta" in the metadata

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 22]
```diff
@@ -0,0 +1,264 @@
...
+* [[!DNL Experience Data Model (XDM)] System](../../../../../xdm/home.md): The standardized framework by which [!DNL Experience Platform] organizes customer experience data.
+  * [Basics of schema composition](../../../../../xdm/schema/composition.md): Learn about the basic building blocks of XDM schemas, including key principles and best practices in schema composition.
+  * [Schema Editor tutorial](../../../../../xdm/tutorials/create-schema-ui.md): Learn how to create custom schemas using the Schema Editor UI.
+* [[!DNL Real-Time Customer Profile]](../../../../../profile/home.md): Provides a unified, real-time consumer profile based on aggregated data from multiple sources.
+
+## Prerequisites {#prerequisites}
+
+In order to connect [!DNL Pinterest Ads] to Experience Platform, you must provide values for the following connection properties:
+
+* The [!DNL Pinterest] `Access Token`.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md]: lower case for access token. (no need for camel case since it is UI side here)

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 23]
```diff
@@ -0,0 +1,264 @@
...
+* The [!DNL Pinterest] `Ad Account ID`.
+* One of [!DNL Pinterest] `Campaign`, `Ad Group` or `Ad` ID's as required.
+
+For more information on each of these, see their individual sections in the [prerequisites](../../../../connectors/advertising/pinterest-ads.md#prerequisites) section of the [!DNL Pinterest Ads] source overview documentation.
+
+### Create Platform Schema {#create-platform-schema}
+
+Create a Platform [schema](/help/xdm/schema/composition.md) required for Linked In Ads.
+![Platform UI screenshot showing an example Schema](../../../../images/tutorials/create/advertising/pinterest-ads/schema.png?lang=en)
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md]: add space between text and screenshot line

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 24]
```diff
@@ -0,0 +1,264 @@
...
+
+You can obtain sample definitions [here](#schema-example).
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md]: "For sample definitions, refer to this [schema example](#schema-example).

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 25]
```diff
@@ -0,0 +1,264 @@
...
+
+## Next steps {#next-steps}
+
+By following this tutorial, you have established a connection to your [!DNL Pinterest Ads] account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](/help/sources/ui-tutorials/dataflow/advertising.md).
+
+## Select data {#select-data}
+
+The **[!UICONTROL Select data]** step appears, providing an interface for you to input the information that would be passed to the APIs to bring the data you want to Platform.
+
+Finally, you must provide as a mandatory input the field values below:
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md:76]: lets add an empty line after this and before the table

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 26]
```diff
@@ -0,0 +1,264 @@
...
+| Field | Description |
+| --- | --- |
+| **[!UICONTROL ad_account_id]** | Your [!DNL Pinterest Ads] ad_account_id. You can refer to the [Find IDs in Ads Manager](https://help.pinterest.com/en/business/article/find-ids-in-ads-manager) page for guidance on how to get the value. |
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md]: "Refer to the [[!DNL Pinterest] guide on finding IDs in Ads Manager]..."

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 27]
```diff
@@ -0,0 +1,264 @@
...
+
+If the details provided are valid, the UI displays a **[!UICONTROL Connected]** status with a green check mark, you can then proceed to the next step.
+
+![Platform UI screenshot for Connect Pinterest Ads account with connected status](../../../../images/tutorials/create/advertising/pinterest-ads/connected.png?lang=en)
+
+## Next steps {#next-steps}
+
+By following this tutorial, you have established a connection to your [!DNL Pinterest Ads] account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](/help/sources/ui-tutorials/dataflow/advertising.md).
+
+## Select data {#select-data}
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md:72]: Let's move this select data step to before the next steps, then we can remove "dataflow detail", "mapping", and "review" since those steps are the modularized in the advertising dataflow tutorial

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 28]
```diff
@@ -0,0 +1,264 @@
...
+
+## Mapping {#mapping}
+
+The [!UICONTROL Mapping] step appears, providing you with an interface to map the source fields from your source schema to their appropriate target XDM fields in the target schema.
+
+Platform provides intelligent recommendations for auto-mapped fields based on the target schema or dataset that you selected. You can manually adjust mapping rules to suit your use cases. Based on your needs, you can choose to map fields directly, or use data prep functions to transform source data to derive computed or calculated values. For comprehensive steps on using the mapper interface and calculated fields, see the [Data Prep UI guide](../../../../../data-prep/ui/mapping.md).
+
+Once your source data is successfully mapped, select **[!UICONTROL Next]**.
+
+![The mapping step of the sources workflow.](../../../../images/tutorials/create/advertising/pinterest-ads/mapping.png)
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md]: these steps can be removed

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 29]
```diff
@@ -0,0 +1,264 @@
...
+
+## Scheduling {#scheduling}
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md:103]: lets move this to additional resources as it is specific to pinterest and has some additional info compared to the modularized scheduling step in the main advertising dataflow tutorial

File: help/sources/tutorials/ui/create/advertising/pinterest-ads.md [hunk 30]
```diff
@@ -0,0 +1,264 @@
...
+| Frequency | Interval |
+| --- | --- |
+| `Day` | 1 |
+| `Hour` | 24 |
+
+Once your have provided the values for your schedule, select **[!UICONTROL Next]**.
+
+![The scheduling step of the sources workflow.](../../../../images/tutorials/create/advertising/pinterest-ads/scheduling.png)
+
+## Review {#review}
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/pinterest-ads.md]: we can remove this step

**Review by carlojavier — CHANGES_REQUESTED** [review]
Hey @mohan-h comments added. please revise, let me know of any questions!

File: help/sources/connectors/advertising/pinterest-ads.md (hunk 14 above)
Comment (mohan-h) [reply help/sources/connectors/advertising/pinterest-ads.md]: Thanks, have reworked the section.

File: help/sources/connectors/advertising/pinterest-ads.md (hunk 1 above)
Comment (mohan-h) [reply help/sources/connectors/advertising/pinterest-ads.md]: Team checked with Pinterest support - the right branding nomenclature is `Pinterest Ads`, which is the one used.
